lib/proofer/vendor/mock_response.rb

```diff
@@ -0,0 +1,8 @@
+require 'hashie'
+
+module Proofer
+  module Vendor
+    class MockResponse < Hashie::Mash
```

Comment:
I would go a step further and use a PORO. OpenStructs are slow and less flexible. http://www.schneems.com/2014/12/15/hashie-considered-harmful.html